jenkins/opensearch-dashboards/dashboards-integ-test.jenkinsfile

```diff
@@ -0,0 +1,376 @@
+/*
```

Comment:
Can you please make changes in current integ-test workflow so that diff is easily visible?

Reply:
I am proposing a dedicated job for OSD integ test because adding this ci-group changes in the existing job will be hacky and not clean. Running ci-groups and different components are technically two different use-cases. 
This can be done in existing file but will make it clunky and hacky.

Reply:
How about putting it in a library and calling the library instead. Seeing a lot of duplicate code and also worried about the maintenance like changing docker image names in both files, agent names, etc.

Reply:
Refactored to use the same job file.